docs/static/jvm.asciidoc

```diff
@@ -5,7 +5,7 @@
 {ls} requires one of these versions:
 
 * Java 11
-* Java 17 (see <<jdk17-upgrade>> for settings info)
+* Java 17 (default, see <<jdk17-upgrade>> for settings info)
```

Comment:
We may want to change https://github.com/elastic/logstash/blame/main/docs/static/jvm.asciidoc#L70

so say `uses` rather than `supports`